routes/skus.rb

```diff
@@ -0,0 +1,121 @@
+# frozen_string_literal: true
+require 'sinatra/base'
+require_relative '../logging/logs'
+
+# Our app
+module Sinatra
+  # Our sku Routing
+  module SkuRoutes
+    # rubocop:disable LineLength, MethodLength, AbcSize, CyclomaticComplexity, PerceivedComplexity
+    def self.registered(app)
+      ##
+      # SKUs Resource
+      ##
+
+      app.get '/skus/new/?:error?' do
+        # get new sku form
+        @error = true if params[:error]
+        @skus = Sku.all
```

Comment:
I dont think this line is needed. (But if it were, you would want to use Client.all. See other comment in create.erb)